controllers/front/CartController.php

```diff
@@ -107,20 +107,24 @@ public function postProcess()
 	protected function processDeleteProductInCart()
 	{
 		$customization_product = Db::getInstance()->executeS('SELECT * FROM `'._DB_PREFIX_.'customization`
-		WHERE `id_product` = '.(int)$this->id_product.' AND `id_customization` != '.(int)$this->customization_id);
+		WHERE `id_cart` = '.(int)$this->context->cart->id.' AND `id_product` = '.(int)$this->id_product.' AND `id_customization` != '.(int)$this->customization_id);
 
 		if (count($customization_product))
 		{
 			$product = new Product((int)$this->id_product);
-
+			if ($this->id_product_attribute>0)
```

Comment:
Hi,

Thank you for your PR.

Would you mind just adding a space before and after the symbol '>'  ?

Thank you. Regards